src/tools/exports/conseiller-zero-permanence.js

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env node
+'use strict';
+const path = require('path');
+const fs = require('fs');
+const { execute } = require('../utils');
+
+execute(__filename, async ({ logger, db }) => {
+  const conseillers = await db.collection('conseillers').find({ statut: 'RECRUTE' }).toArray();
+  let promises = [];
+  let count = 0;
+  logger.info(`Generating CSV file...`);
+  let csvFile = path.join(__dirname, '../../../data/exports', 'conseiller-zero-permanence.csv');
+
+  let file = fs.createWriteStream(csvFile, {
+    flags: 'w'
+  });
+
+  // eslint-disable-next-line max-len
+  file.write('id du conseiller; Nom; Prenom;Email perso du conseiller (saisi lors de son inscription);email @conseiller-numerique.fr;Email Pro saisi dans l\'espace Coop;Téléphone perso (saisi avant d\'etre recruté);Téléphone Pro (saisi dans l\'espace coop);conseiller coordinateur;Espace COOP activé;Nom SA;Siret SA;SA email;SA téléphone\n');
+  conseillers.forEach(cnfs => {
+    promises.push(new Promise(async resolve => {
+      const permanences = await db.collection('permanences').countDocuments({ conseillers: { '$in': [cnfs._id] } });
```

Comment:
ok normalement on devrait avoir la même avec le test hasPermanence à true dans la collection conseillers ?

Reply:
je crois que y a un 2ème array conseillersItinerants (un truc comme ca)
cela voudrait dire de mettre un $or de $in soit de conseillers soit de conseillersItinerants

Reply:
Non c'est forcément dans conseiller et itinérant si il clique sur la case à cocher